examples/over-sampling/plot_over_sampling_benchmark_3nn.py

```diff
@@ -0,0 +1,97 @@
+"""
+=============================================================
+Benchmark over-sampling methods using a 3NN classifier
+=============================================================
+In this computer vision example two faces are used from the LFW 
+(Faces in the Wild) dataset. Several implemented over-sampling 
+methods are used in conjunction with a 3NN classifier in order 
+to examine the improvement of the classifier's output quality 
+by using an over-sampler.
+"""
+print(__doc__)
+
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy import interp
+from sklearn import datasets, naive_bayes, neighbors
+from sklearn.metrics import auc, roc_curve
+from sklearn.model_selection import StratifiedKFold
+
+from imblearn.over_sampling import ADASYN, SMOTE, RandomOverSampler
+from imblearn.pipeline import make_pipeline
+
+RANDOM_STATE = np.random.RandomState(0)
+
+
+class DummySampler(object):
+
+    def sample(self, X, y):
+        return X, y
+
+    def fit(self, X, y):
+        return self
+
+    def fit_sample(self, X, y):
+        return self.sample(X, y)
+
+
+cv = StratifiedKFold(n_splits=5)
+
+# Load the dataset
+data = datasets.fetch_lfw_people()
+majority_person = 1871  # 530 photos of George W Bush
+minority_person = 531  # 29 photos of Bill Clinton
+idx = np.where((data.target == majority_person) |
```

Comment:
I would probably use `np.flat_nonzero` to avoid `[0]` and `np.bitwise_or()` instead of `|`.
I find it easier to read. What do you think?